src/context_store/orchestrator.py

```
@@ -487,6 +492,13 @@ async def create_orchestrator(
             settings=settings,
         )
 
+        # BatchProcessor 組み立て（task_registry は LifecycleManager と共有）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)